src/timedisc/timedisc.jl

```diff
@@ -1,4 +1,155 @@
 
+# The following structures and methods provide a minimal implementation of
+# the low-storage explicit Runge-Kutta method of
+# Carpenter, Kennedy (1994) Fourth order 2N storage RK schemes, Solution 3
+# using the same interface as OrdinaryDiffEq.jl.
+struct SimpleAlgorithm2N45
+  a::SVector{5, Float64}
+  b::SVector{5, Float64}
+  c::SVector{5, Float64}
+
+  function SimpleAlgorithm2N45()
+    a = @SVector [0.0, 567301805773.0 / 1357537059087.0,2404267990393.0 / 2016746695238.0,
+       3550918686646.0 / 2091501179385.0, 1275806237668.0 / 842570457699.0]
+    b = @SVector [1432997174477.0 / 9575080441755.0, 5161836677717.0 / 13612068292357.0,
+        1720146321549.0 / 2090206949498.0, 3134564353537.0 / 4481467310338.0,
+        2277821191437.0 / 14882151754819.0]
+    c = @SVector [0.0, 1432997174477.0 / 9575080441755.0, 2526269341429.0 / 6820363962896.0,
+        2006345519317.0 / 3224310063776.0, 2802321613138.0 / 2924317926251.0]
+
+    new(a, b, c)
+  end
+end
+
+mutable struct SimpleIntegrator2NOpts{Callback}
+  callback::Callback
+  adaptive::Bool
+  dtmax::Float64
+  tstops::Vector{Float64}
+end
+
+function SimpleIntegrator2NOpts(callback, tspan; kwargs...)
+  SimpleIntegrator2NOpts{typeof(callback)}(
+    callback, false, Inf, [last(tspan)])
+end
+
+mutable struct SimpleIntegrator2N{RealT<:Real, uType, Params, Sol, Alg, SimpleIntegrator2NOpts}
+  u::uType
+  du::uType
+  u_tmp::uType
+  t::RealT
+  dt::RealT
+  dtcache::RealT
+  iter::Int
+  p::Params
+  sol::Sol
+  alg::Alg
+  opts::SimpleIntegrator2NOpts
+  finalstep::Bool
+end
+
+function solve(ode::ODEProblem, alg::SimpleAlgorithm2N45;
```

Comment:
`solve` needs some (high-level) documentation on its purpose and what it does (i.e., "creates storage, creates integrator" etc.)

Reply:
Done in a0ca542.